docs/tutorial/notebook/reading_a_dataset.ipynb

```diff
@@ -0,0 +1,373 @@
+{
```

Comment:
<p>Only keep the part of the image inside the red box - that's clearer than the entire image + arrow.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/isl-org/Open3D-ML/pull/512/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>Done.</p>